### DISCUSSION TITLE
Hi, does anyone knows you we can enable premium logo branding?

### DISCUSSION THREAD

**Discussion by Ionutovic** [arichornlover/uYouEnhanced#573] [Q&A]
Hi, does anyone knows you we can enable premium logo branding? On the other versions there was a setting, I don't remember the name exactly, something like: enablePremiumBranding but now it seems to be gone.

**Comment by arichornlover** [comment]
`Fake Premium` is an option in uYouEnhanced Settings, mind trying that?

**Reply by Ionutovic** [reply]
> `Fake Premium` is an option in uYouEnhanced Settings, mind trying that?

I'm looking for the same thing, there used to be an option but seems gone in the latest version.

**Reply by arichornlover** [reply]
Than I don’t know, maybe that option (or A/B Flag) was removed by Google?

**Reply by Ionutovic** [reply]
> Than I don’t know, maybe that option (or A/B Flag) was removed by Google?

Honestly I don't know how the app works but the option was in AB settings.
